build.gradle

```diff
@@ -1,27 +1,18 @@
-// **********************************************************************
 //
-// Copyright (c) 2014-present ZeroC, Inc. All rights reserved.
+// Copyright (c) ZeroC, Inc. All rights reserved.
```

Comment:
let's keep the ZeroC header, the plan is still to submit that upstream

Reply:
Looking at the master branch, they've changed to the format used here 
https://github.com/zeroc-ice/ice-builder-gradle/blob/master/build.gradle